packages/calcite-components/src/components/switch/switch.scss

```diff
@@ -52,60 +71,69 @@
 }
 
 .track {
-  @apply bg-foreground-2
-    border-color-2
-    pointer-events-none
+  @apply pointer-events-none
     relative
     box-border
     inline-block
-    rounded-full
     border
     border-solid
     align-top
     focus-base;
+
+  // todo: need var for 9999px?
+  border-radius: var(--calcite-switch-corner-radius, 9999px);
+  border-color: var(--calcite-switch-border-color, var(--calcite-color-border-2));
+  background-color: var(--calcite-switch-background-color, var(--calcite-color-foreground-2));
 }
 
 .container:focus .track {
   @apply focus-outset;
 }
 
 .handle {
-  @apply bg-foreground-1
-    border-color-input
-    pointer-events-none
+  @apply pointer-events-none
     absolute
     block
-    rounded-full
     border-2
     border-solid
     transition-all
     duration-150
     ease-in-out;
   inset-block-start: -1px;
   inset-inline: -1px theme("inset.auto");
+  background-color: var(--calcite-switch-handle-background-color, var(--calcite-color-foreground-1));
+  border-color: var(--calcite-switch-handle-border-color, var(--calcite-color-border-input));
+  border-radius: var(--calcite-switch-handle-corner-radius, 9999px);
+  box-shadow: var(--calcite-switch-handle-shadow);
 }
 
 .container:hover,
 .container:focus {
+  .track {
+    background-color: var(--calcite-switch-background-color-hover);
+  }
   .handle {
-    @apply border-color-brand;
-    box-shadow: inset 0 0 0 1px var(--calcite-color-brand);
+    border-color: var(--calcite-switch-handle-border-color, var(--calcite-color-brand));
+    background-color: var(--calcite-switch-handle-background-color-hover);
+    // todo: do we need a var for `inset 0 0 0 1px var(--calcite-color-brand)`?
+    box-shadow: var(--calcite-switch-handle-shadow, inset 0 0 0 1px var(--calcite-color-brand));
   }
 }
 
 :host([checked]) {
   .track {
-    @apply bg-brand border-color-brand-hover;
+    border-color: var(--calcite-switch-border-color, var(--calcite-color-brand-hover));
+    background-color: var(--calcite-switch-background-color-hover, var(--calcite-color-brand));
   }
   .handle {
-    @apply border-color-brand;
+    border-color: var(--calcite-switch-handle-border-color, var(--calcite-color-brand));
     inset-inline: theme("inset.auto") -1px;
   }
 
   .container:hover {
```

Comment:
update this to `&:host(:hover)`

Reply:
Something to watch out for here, is that we also need to check for disabled now. `&:host(:hover:not([disabled])) {`

Because previously the container was inside the component's InteractionContainer with pointer-events none. Moving this selector to the host means we need to check to make sure it doesn't apply when disabled.